packages/web-components/fast-components/src/radio/radio.md

```diff
@@ -0,0 +1,109 @@
+# Radio
+
+## Overview
+An implementation of a radio button as a form-connected web-component. Facilitating single select from a group of visible choices.
+
+### Use Cases
+Used anywhere an author might otherwise use an `input[type="radio"]`. Used to facilitate choice where only one choice is acceptable.
+  
+### Features
+- form association
+- focus delegation
+
+### Risks and Challenges
+We want general feature-parity between this component and an `input[type="radio"]`.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/radio)
+- [Material UI](https://material-ui.com/components/radio-buttons/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/radio-group/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/radio-button/code)
+- [Ant Design](https://ant.design/components/radio/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/radio)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/radio-button)
+---
+
+### API
+Extends [form associated custom element](../form-associated/form-associated-custom-element.md).
+
+*Component Name*
+`fast-radio`
+
+*IDL attributes*
+- `checked: boolean`
+  - The current checked state of the radio
+
+*Content attributes*
+- `readonly`
+  - The radio should be submitted with the form but should not be editable.
+- `disabled`
+  - The radio should be disabled from user interaction and will not be submitted with the form data.
+- `value` - Not visible to the user, it's used for form data and to distinguish between other radio buttons of the same name attribute value.
+- `name` - All radio buttons with the same name will operate as a group allowing only 
+one radio within the group to be selected at a time.
+- `checked`
+  - The initial checked value. 
+  
+*Events*
+- `change: CustomEvent`
+  - no custom data
+  - bubbles
+
+### Anatomy and Appearance
+
+```HTML
+<!-- shadow root -->
+<template 
+  role="radio"
```

Comment:
you'll likely need disabled to map to aria-disabled here.

Reply:
All the aria stuff was documented in the accessibility section but I suppose if I put role here I should put the rest or just take out the template part here and leave the aria documentation in the other section?

Reply:
I think it can exist in both - they serve different purposes. With it left off here it seems to be missing. If I skip this and go down to accessibility I'd expect what you have outlined there. Since you're showing an example of the actual template, I'd expect it to be called out here specifically I think.

Reply:
removed template from here and left documentation in accessibility section. Hopefully these days it's assumed that would go in the <template/> bit but I can change if you disagree

Reply:
I'm not going to push on it - I just tend to write these as though I'm not going to be actually doing the work (which is what happened with tooltip) :).